daemon/openrazer_daemon/keyboard.py

```diff
@@ -64,44 +64,45 @@
 
 }
 
-#      0   1   2   3   4    5   6   7
-#    ------------------------------------
-# 0 |  1   2   3   4   5        UP
-# 1 |  6   7   8   9   10  LEFT    RIGHT
-# 2 |  11  12  13  14  15  TMB DOWN MS
-# 3 |  16  17  18  19  20
+#      0   1   2   3   4    5   6   7   8
+#    ---------------------------------------
+# 0 |
+# 1 |      1   2   3   4   5        UP
+# 2 |      6   7   8   9   10  LEFT    RIGHT
+# 3 |      11  12  13  14  15  TMB DOWN MS
+# 4 |      16      17  18  19  20
 #
 
 ORBWEAVER_KEY_MAPPING = {
-    '1': (0, 0),
-    '2': (0, 1),
-    '3': (0, 2),
-    '4': (0, 3),
-    '5': (0, 4),
-    'UP': (0, 6),
```

Comment:
This would break the normal Orbweaver then. Please add a new new section (`ORBWEAVER_CHROMA_KEY_MAPPING`) and then adjust the code in `pylib/openrazer/client/macro.py` and `daemon/openrazer_daemon/misc/key_event_management.py` to use that *just* for the Orbweaver Chroma

Reply:
What would it break for the Orbweaver?

Reply:
Reverted, as this currently has no bearing on reality